kitsune/wiki/tests/test_templates.py

```diff
@@ -192,13 +192,27 @@ def test_document_fallback_with_translation_english_slug(self):
 
     def test_document_fallback_no_translation(self):
         """The document template falls back to English if no translation
-        exists."""
+        exists. The fallback message is not shown."""
```

Comment:
The doc-string should be `The fallback message is only shown when there is any revision ready for localization`